packages/apollo-gateway/src/buildQueryPlan.ts

```diff
@@ -768,6 +784,7 @@ export class QueryPlanningContext {
   getKeyFields(
```

Comment:
I think we should rewrite this to use named arguments (i.e. an object to destructure) so it is clearer at its call site what is being done with the `fetchAll` argument

Reply:
done! (thanks tsc)